exonum-java-binding/core/rust/integration_tests/src/vm.rs

```diff
@@ -184,10 +184,10 @@ fn project_root_dir() -> PathBuf {
 
 #[cfg(debug_assertions)]
 fn target_path() -> &'static str {
-    "target/debug"
+    "target/debug/deps"
```

Comment:
Please document why these directories are used, with appropriate documentation links if needed — it is not apparent.